spec/features/toggle_language_spec.rb

```diff
@@ -0,0 +1,216 @@
+describe 'ToggleLanguage', js: true do
+  let(:user) { create :user1 }
+
+  after(:each) do
+    page.reset!
```

Comment:
Maybe add a comment here that mentions this clears localStorage in order to prevent test pollution?